pkg/model/network.go

```diff
@@ -120,9 +120,8 @@ func (b *NetworkModelBuilder) Build(c *fi.ModelBuilderContext) error {
 			VPC:         b.LinkToVPC(),
 			DHCPOptions: dhcp,
 		})
-	} else {
-		// TODO: would be good to create these as shared, to verify them
 	}
+	// TODO: would be good to create these as shared, to verify them
```

Comment:
I think putting this TODO comment above the `if !sharedVPC {` conditional that it was originally in might be more appropriate.

Reply:
Sure.this seems good to me.
I will update it.

Reply:
ah, I meant above line 101. sorry. given theres no `else` case anymore, a comment related to an `else` case would probably be best above the `if` statement itself.